bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/messages/Messages.properties

```
@@ -0,0 +1,90 @@
+mvn_int_test_report_none_found=No integration test results were found for the {0} project. Select "{1}" before you select "{2}" on the menu.
+mvn_int_test_report_general_error="An error was detected when the view integration test report request was processed on the {0} project.
+
+mvn_unit_test_report_no_project_found=An error was detected when the view unit test report request was processed. The object that represents the selected project could not be found. When you use the Run Configuration launcher, be sure to select a project or project content first.
+mvn_unit_test_report_none_found=No unit test results were found for the {0} project. Select "{1}" before you select "{2}" on the menu.
+mvn_unit_test_report_general_error=An error was detected when the view unit test report request was processed on the {0} project.
+
+gradle_test_report_no_project_found=An error was detected when the view test report request was processed. The object that represents the selected project could not be found. When you use the Run Configuration launcher, be sure to select a project or project content first.
+gradle_test_report_none_found=No test results were found for the {0} project. Select "{1}" before you select "{2}" on the menu.
+gradle_test_report_general_error=An error was detected when the view test report request was processed on the {0} project.
+
+issue_stop_prompt=Would you like to issue the Liberty Maven or Gradle stop command for this project to stop a Liberty server that might still be running the project outside of the Liberty Tools session?
+plugin_stop_issue_error=An exception occurred when the plugin stop command was issued.
+plugin_stop_timeout=The Liberty Maven or Gradle stop command issued for the {0} project timed out after {1} seconds.
+plugin_stop_failed=The stop command failed with the following exitValue: {0}
+plugin_stop_general_error=An error was detected when the Liberty Maven or Gradle stop command was processed on the {0} project.
+
+# WorkspaceProjectsModel
+project_analyze_error=An error occurred when the projects in the workspace were analyzed.
+
+# DashboardView
+project_not_gradle_or_maven=The {0} project is not a Gradle or Maven project.
+image_descriptions_error=An error was detected when the image descriptions were retrieved.
+action_general_error=An error was detected during the {0} action.
+dashboard_refresh_error=An error was detected when the Liberty dashboard content was refreshed.
+
+# JRETab
+java_default_set_error=Unable to set the default Java installation that was obtained from the build path of the {0} project in the {1} configuration.
+java_resolution_error=Unable to resolve the Java installation path by using the {0} configuration. The project will use the workspace Java installation instead.
```

```suggestion
java_resolution_error=Unable to resolve the Java installation path by using the {0} configuration. The project uses the workspace Java installation instead.
```

per IBM style, avoid future tense